src/Compiler/CompilerPasses.cpp

```diff
@@ -201,6 +200,10 @@ void addONNXToKrnlPasses(mlir::PassManager &pm, int optLevel, bool enableCSE,
 void addKrnlToAffinePasses(mlir::PassManager &pm) {
   pm.addNestedPass<func::FuncOp>(
       onnx_mlir::krnl::createConvertKrnlToAffinePass());
+  if (enableParallel) {
```

Comment:
Could you put a comment here to say the purpose of calling these passes? It helps if someone tries to reorganize the passes.

Reply:
make sense, done.